website/content/v1.7/talos-guides/install/cloud-platforms/aws.md

```diff
@@ -7,52 +7,102 @@ aliases:
 
 ## Creating a Cluster via the AWS CLI
 
-In this guide we will create an HA Kubernetes cluster with 3 worker nodes.
-We assume an existing VPC, and some familiarity with AWS.
+In this guide we will create an HA Kubernetes cluster with 3 control plane nodes across 3 availability zones.
```

Comment:
I was always confused by spreading the etcd across AZes:

* increased latency
* a chance of network partitioning

As long as you don't have 3x workers spread across AZes same way, what kind of benefit does it give to the availability of the cluster?

Reply:
AZ spread is recommended for all AWS HA architecture. Single AZs have problems more often than people realize. All AZs are supposed to be in "single digit" ms latency between locations (Azure and GCP don't guarantee that). I know this is a getting started guide, but we should still recommend some HA configurations for the components that matter.

Reply:
So it's not that I'm trying to say that spreading CPs across AZs is a bad idea, but I want to make sure that users understand what gets actually protected and what does not by spreading CPs across AZs.

Reply:
This version of the doc didn't have the autoscaling group. I just updated it with the correct version. Workers are created in an ASG and if an AZ goes away the ASG will re-balance nodes or create new nodes in AZs that are available.